index.html

```diff
@@ -162,5 +162,58 @@ <h2 class="about_subtitle">Skills</h2>
         <img class="about_icon" src="/icons/bulb.svg" alt="light bulb">
       </div>
     </section>
+    <section id="form">
+      <div class="left-form">
+        <img src="icons/figure1.png" class="figure1" alt="triangle">
+        <img src="icons/figure2.png" class="figure3" alt="rhombus">
+      </div>
+      <h2 class="form_title">Get Started</h2>
+      <p class="form-par">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      <form action="https://formspree.io/f/xgerpyrn" method="POST">
+        <ul>
+          <li class="inside">
+            <label class="text">Your name</label>
+            <input type="text" name="name" maxlength="30" placeholder="e.g. Javier Rodriguez" required> 
+          </li>
+          <li class="inside">
+            <label class="email">Your email</label>
+            <input type="email" name="email" placeholder="e.g. example@example.com" required>
+          </li>
+          <li class="textarea">
+            <textarea name="text" maxlength="500" rows="10" placeholder="Enter your message here..." required></textarea>
+          </li>
+          <li class="button">
+            <button type="submit" class="form-btn">
+              Start Collaboration
+            </button>
+          </li>
+        </ul>
```

Comment:
### OPTIONAL
- [ ] You don't have to wrapped by list when you create a form. [form-elements](https://www.w3schools.com/html/html_forms.asp)